app.js

```diff
@@ -24,10 +24,35 @@ const projectName = "food-wars";
 
 app.locals.appTitle = `${capitalized(projectName)} created with IronLauncher`;
 
+const session =require('express-session');
+const mongoStore = require('connect-mongo');
+const mongoose = require('mongoose');
+
+app.use(
+    session({
+        secret: process.env.SESSION_SECRET,
+        resave: false,
+        saveUninitialized: false,
+        cookie: {
+            maxAge: 1000*60*60*24,
+        },
+        store: new mongoStore({
+            mongoUrl: 'mongodb://0.0.0.0/food-wars',
+            })
+    })
+)
+
```

Comment:
In day 7 of module 2 we did it in the folder config/session.config > can we use the same approach?